src/quo2/components/avatars/account_avatar.cljs

```diff
@@ -0,0 +1,50 @@
+(ns quo2.components.avatars.account-avatar
+  (:require [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo.theme :as theme]))
+
+(def icon-color-value
+  {:dark (get-in colors/customization [:dark :purple])
+   :light (get-in colors/customization [:light :purple])})
+
+(def sizes
+  {:xxl 80
+   :xl 48
+   :large 32
+   :medium 24
+   :small 20})
+
+(def border-radii
+  {:xxl 16
+   :xl 12
+   :large 10
+   :medium 8
+   :small 6})
+
+(def inner-icon-sizes
+  {:xxl 36
+   :xl 24
+   :large 15
+   :medium 11
+   :small 11})
+
+(defn account-avatar
+  [{:keys [size icon] :or {size :xxl
+                           icon :main-icons/placeholder}}]
+  (let [icon-color (if (theme/dark?)
+                     (:dark icon-color-value)
+                     (:light icon-color-value))
+        avatar-size (get sizes size)
+        avatar-border-radius (size border-radii)
+        inner-icon-size (size inner-icon-sizes)]
```

Comment:
my comment was not only about sizes but for all maps avatar-border-radius and inner-icon-size as well

Reply:
Okay, Doing now

Reply:
Done, Applied Parvesh's refactor ..